prime-router/src/main/kotlin/tokens/Server2ServerAuthentication.kt

```diff
@@ -148,15 +122,11 @@ class TokenAuthentication : Logging {
      *
      * @return the authenticated JWT Claims set if authentication was both successful.  Otherwise returns null.
      */
```

Comment:
This is a key change.  The old code was returning a server2server specific claims obj; the new code is returning our AuthenticatedClaims obj, which completely hides whether the authentication was done via Okta or Server2server.